hack/ci/e2e.sh

```diff
@@ -187,9 +187,45 @@ EOF
         | grep -cv "node-role.kubernetes.io/master" \
     )"
 
+    KUBERNETES_PRESUBMIT_JOB=true
     # ginkgo regexes
-    SKIP="${SKIP:-}"
-    FOCUS="${FOCUS:-"\\[Conformance\\]"}"
+    if [[ "${KUBERNETES_PRESUBMIT_JOB:-}" = true ]]; then
+        # k/k presubmit jobs regex
+        SKIP="${SKIP:-"\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]"}"
+        FOCUS="${FOCUS:-"\\[Slow\\]"}"
+        # skip tests that are not able to rung with kind
```

Comment:
rung -> run

Reply:
this was just only for testing, the important one is the job that Ben is setting up in test-infra

Reply:
Yeah, this is one of the ways we determined the regex (though not all of it, it doesn't match what were actually using for now), we opted to manage them with the jobs instead to better handle different Kubernetes branches etc.

The regex will change as we manage to improve the tests upstream.

Reply:
(in addition to kind changes that would affect all branches)